README.md

```diff
@@ -42,6 +42,19 @@ Fetch the API schema to confirm the server is up and running. (This doesn't requ
 
     curl http://localhost:8080/v3/api-docs.yaml
 
+### Running the server via `docker`
+
+If you wish to run the Terraware server and/or Postgres database via docker, check out the `./docker/Makefile`.
+
+#### Run both the server and database
+This is useful if you just want the server running locally for frontend development
+1. Copy `./docker/.env.sample` to `./docker/.env` and fill in the necessary Keycloak environment variables
+2. Run `cd docker && make run`
+
+### Run just the database
+This is useful if you are doing backend development and running the server in IntelliJ, and you don't want to install Postgres natively
+1. Run `cd docker && make run-postgres`
```

Comment:
`make -C docker run-postgres` would also work (main difference is that the shell will stay pointed at the repo root directory afterwards).